ecosystem/sep-0006.md

```diff
@@ -605,6 +625,7 @@ The JSON object contains an entry for each Stellar asset that the anchor support
 
 * `enabled`: `true` if SEP-6 withdrawal for this asset is supported
 * `authentication_required`: Optional. `true` if client must be [authenticated](#authentication) before accessing the deposit endpoint for this asset. `false` if not specified.
+* `quotes_supported`: Optional. `true` if client can use [SEP-38] quotes to perform a bridge between non-equivalent assets, for instance by withdrawing fiat BRL from Stellar-USDC. `false` if not specified.
```

Comment:
❓  Do we want to use `quote_required` vs `quotes_supported`? Or should we have both attributes?

Imagine an anchor operates in a country that only has one fiat currency (ex. the US for USD). In this case anchoring another Stellar asset would require `quote_required` since thats the _only_ method of receiving that asset. 

But imagine an anchor operates in a country that has multiple fiat currencies (or multiple state-recognized / bank-supported currencies). In this case `quotes_supported` should be used because the user _could_ want to exchange non-equivalent assets OR the user could just want to exchange equivalent assets. Thoughts?

Reply:
👏  I really like the idea of having both `quote_required` and `quotes_supported` parameters since it would make the implementation clearer as developers won't need to make any assumptions nor read between the lines.

---

> Imagine an anchor operates in a country that only has one fiat currency (ex. the US for USD). In this case anchoring another Stellar asset would require quote_required since thats the only method of receiving that asset.

Even in the US we could have an Anchor serving both `USD <> USDC` and `ETH <> Stellar-ETH` for instance, and that same anchor could decide to make `USD <> Stellar-ETH`.

In addition to that, with `quote_required` we can have anchors focused only on making `USD <> Stellar-ETH`, which sounds like a valuable use case.

Reply:
Added the `quotes_required` parameter in https://github.com/stellar/stellar-protocol/pull/973/commits/ef5c902b7f4a1690d95322df0c5c7bd579a78099.